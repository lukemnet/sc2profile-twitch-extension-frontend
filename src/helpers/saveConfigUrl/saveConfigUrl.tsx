import getApiUrl from 'helpers/getApiUrl/getApiUrl';

const saveConfigUrl = (channelId: string) =>
  getApiUrl({ channelId, type: 'config', activity: 'save' });

export default saveConfigUrl;
