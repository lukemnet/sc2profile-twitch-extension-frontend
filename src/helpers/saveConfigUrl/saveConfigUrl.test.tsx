import saveConfigUrl from 'src/helpers/saveConfigUrl/saveConfigUrl';

jest.mock('src/helpers/getApiUrl/getApiUrl', () => {
  return jest.fn().mockImplementation(() => ({
    url: 'testSaveConfigUrl',
    method: 'TEST_SAVE_CONFIG_METHOD',
  }));
});


it('returns correct save config API URL', () => {
  expect(saveConfigUrl('123')).toMatchSnapshot();
});