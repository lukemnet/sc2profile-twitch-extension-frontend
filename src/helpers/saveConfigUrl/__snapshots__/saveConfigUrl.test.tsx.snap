// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns correct save config API URL 1`] = `
Object {
  "method": "TEST_SAVE_CONFIG_METHOD",
  "url": "testSaveConfigUrl",
}
`;
